### DISCUSSION TITLE
请问如何设置预览的默认 render

### DISCUSSION THREAD

**Discussion by yimiaoxiehou** [AlistGo/alist#3460] [General]
如题，我想将 html 的 默认 review 设置为  html render（现为 markdown），但是找不到设置点，貌似设置外部预览可以，但是外部预览配置的 html render js 地址在哪？示例只有 epub 的

**Comment by xx6z8v** [comment]
到管理-设置-预览-文本类型,把html删除就行,不过会有Bug
